test/Microsoft.AspNet.Server.KestrelTests/EngineTests.cs

```diff
@@ -54,6 +56,14 @@ ILibraryManager LibraryManager
             }
         }
 
+        ILoggerFactory LoggerFactory
+        {
+            get
+            {
+               return new TestLoggerFactory();
```

Comment:
This is still returning a new instance every time - that is highly unusual for properties because it means if code uses this property twice, it gets a different value. Recommend directly using either `NullLogger`/`Factory` or `TestLogger`/`Factory` from https://github.com/aspnet/Testing/tree/dev/src/Microsoft.AspNet.Testing.Logging instead anyway.